api/spec/v1.yaml

```diff
@@ -1533,6 +1533,8 @@ components:
         - roothash.execution_discrepancy
         - roothash.executor_committed
         - roothash.finalized
+        - roothash.message
```

Comment:
wow didn't know this was still in our codebase

Reply:
which thing?

Reply:
that there's still a MessageEvent type in oasis-core. is it even used anywhere? i.e. in the current eden+ version.

Reply:
I can't tell for sure. It's included in the [RoundResults struct](https://github.com/oasisprotocol/oasis-core/blob/fba5c6cc0b2038860333eb09a4c7ccf560150c70/go/roothash/api/results.go#L5-L10), which is exposed via the [Get]LastRoundResults, as mentioned. I'm less clear on where that is called; it's harder to track when RPCs are involved.

The only place I see at a glance is for generating a genesis file, possibly so that messages are not lost during dump-restore: https://github.com/oasisprotocol/oasis-core/blob/fba5c6cc0b2038860333eb09a4c7ccf560150c70/go/consensus/cometbft/apps/roothash/genesis.go#L66-L66

Anyway, what motivated the diff we are commenting on? Just Cobalt?

Reply:
oh the round results, that must be it 

yeah it's for cobalt

Reply:
oops wrong account, gonna repost

Reply:
(reposting, wrong account last time)

oh the round results, that must be it

yeah this addition is for cobalt

Reply:
alright adding a comment about this